CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
   set(CMAKE_MODULE_LINKER_FLAGS "/SAFESEH:NO /NODEFAULTLIB:LIBC")
   add_compile_definitions(IS_WINDOWS _CRT_SECURE_NO_WARNINGS WIN32 _CRT_SECURE_NO_WARNINGS _CRT_SECURE_NO_DEPRECATE _CRT_NONSTDC_NO_DEPRECATE)
 
-  set(PLATFORM_INCLUDES "lib/win/DirectX" "lib/win")
+  set(PLATFORM_INCLUDES "lib/win/DirectX" "lib/win" ${SDL2_INCLUDE_DIR} ${SDL2_INCLUDE_DIRS})
```

Comment:
I think this is redundant. `SDL2::SDL2` already adds includes to relevant targets.

Reply:
The Windows CI builds failed without it

Reply:
linking to a target does not provide includes

Reply:
but they're not platform includes anymore, all platforms should include SDL2 headers now

Reply:
That's because you've added SDL.h header to `linux/linuxdata.cpp`. Add `SDL2::SDL2` to `linux`'s `target_link_libraries` and all will be set.

Reply:
aha, i see